### DISCUSSION TITLE
Do AIR apps launch faster with a license?

### DISCUSSION THREAD

**Discussion by BladePoint** [airsdk/Adobe-Runtime-Support#2348] [Q&A] [answered]
I'm currently building my desktop AIR app on the free tier, so the Harman splash screen is displayed for about 2 seconds before it starts. I understand that if I buy a license I will have the ability to remove the splash screen. Does that mean my app will launch 2 seconds faster?

**Comment by ajwfrost** [comment, marked as answer]
There's a little bit of parallel activity going on - we tried to get the splash screen displayed as soon as the bootstrap code has run, and then it stays there whilst the main SWF is loading and being parsed, then once it's pretty much ready, we trigger the removal (but with a 2s minimum time there). It changes a little from platform to platform though.

But essentially, yes things should start quicker if you don't have the splash screen.

See also https://airsdk.harman.com/pricing

thanks
